language/tools/move-cli/src/package/cli.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    fs::create_dir_all,
+    io::Write,
+    path::{Path, PathBuf},
+    time::Instant,
+};
+
+use anyhow::Result;
+use codespan_reporting::term::termcolor::{ColorChoice, StandardStream};
+
+use move_package::source_package::layout::SourcePackageLayout;
+use move_prover::run_move_prover_with_model;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub enum PackageCommand {
+    /// Create a new Move package with name `name` at `path`. If `path` is not provided the package
+    /// will be created in the directory `name`.
+    #[structopt(name = "new")]
+    New {
+        /// The name of the package to be created
+        name: String,
+    },
+    /// Build the package at `path`
+    #[structopt(name = "build")]
+    Build,
+    /// Generate error map for the package and its dependencies at `path`
+    #[structopt(name = "errmap")]
+    ErrMapGen {
+        /// The prefix that all error reasons will be prefixed with, e.g., "E"
+        error_prefix: Option<String>,
```

Comment:
Sorry if I'm missing something obvious, but, what use cases do you anticipate for this feature? Maybe examples could be included in the docstring?

Reply:
This is used for generating error mappings that you can use with the explanation tool. e.g., instead of `MoveAbort(4096)` you can use this error map to get human readable explanations. I'll add some more to the docstring :)